data/Templates/eCR/ValueSet/_SystemReference.liquid

```diff
@@ -1,40 +1,37 @@
 {%- comment -%} See https://www.hl7.org/fhir/terminologies.html#system for finding the corresponding URI for a system OID {%- endcomment -%}
-{%- assign url = '""' -%}
+{%- assign url = code -%}
 {%- if code -%}
     {%- assign s = code | match: "^([0-2])((\.0)|(\.[1-9][0-9]*))*$" | size -%}
     {%- if s == 1 -%}
```

Comment:
are these `if`s still needed? seems like there wouldn't be any harm in moving the `case code` to the top level of this logic, maybe make the top assignment `code | default: ""` to be safe that it isn't null?